use-timescale/ha-replicas/read-scaling.md

```diff
@@ -8,71 +8,110 @@ tags: [replicas, scaling, ha]
 
 # Manage read replication
 
-You use read replicas to power your read-intensive apps and business intelligence tooling. Using read replicas to serve 
-reads for your app removes load from the primary data instance, and enables your service to improve ingest performance. 
+You use $READ_REPLICAs to power your read-intensive apps and business intelligence tooling. Using $READ_REPLICAs to serve 
+reads for your app removes the load from the primary data instance, and enables your $SERVICE_SHORT to improve ingest performance. 
 This is particularly useful when read traffic is very spiky and risks impacting ingest performance, or where reads have 
 a lower priority to writes. 
 
-This page shows you how to create and manage read replicas.
+This page shows you how to create and manage $READ_REPLICAs.
 
 ## What is read replication?
 
-A read replica is a read-only copy of the primary data instance in your Timescale Cloud service. Queries on read 
-replicas have minimal impact on the performance of the primary data instance. This enables you to interact with 
-up-to-date production data for analysis or to scale out reads beyond the limits of your primary data instance. You use 
-read replicas for read scaling. To limit data loss for your Timescale Cloud services, use [High availability][ha].
+A $READ_REPLICA is a read-only copy of the primary data instance in your $SERVICE_LONG. Queries on $READ_REPLICAs have minimal impact on the performance of the primary data instance. This enables you to interact with 
+up-to-date production data for analysis or to scale out reads beyond the limits of your primary data instance. Read replicas can be short-lived and deleted when a session of data analysis is complete, or long-running to power a
+business intelligence (BI) tool. 
```

Comment:
both use cases are about data analysis. maybe the second example could be about serving app servers instead?

```suggestion
or long-running to serve read traffic from your application servers.
```